app/src/main/java/org/mozilla/focus/notification/RocketMessagingService.java

```diff
@@ -0,0 +1,43 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.notification;
+
+import android.app.PendingIntent;
+import android.content.Intent;
+import android.net.Uri;
+import android.support.v4.app.NotificationCompat;
+
+import org.mozilla.focus.activity.MainActivity;
+
+public class RocketMessagingService extends FirebaseMessagingServiceWrapper {
+
+    @Override
+    public void onRemoteMessage(String url, String title, String body) {
```

Comment:
onMessageReceived is implemented in FirebaseMessagingServiceWrapper ( if it's actual implementation).
And onRemoteMessage() will be called in there.